hibernate6/spanner-hibernate6-dialect/src/main/java/com/google/cloud/spanner/hibernate/SpannerServiceContributor.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2019-2020 Google LLC
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+package com.google.cloud.spanner.hibernate;
+
+import com.google.cloud.spanner.hibernate.schema.SpannerSchemaManagementTool;
+import java.util.Map;
+import java.util.Objects;
+import org.hibernate.boot.registry.StandardServiceInitiator;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+// import org.hibernate.hql.spi.id.inline.InlineIdsOrClauseBulkIdStrategy;
```

Comment:
* All commented code can be removed.
* Did our pre-submit catch such lint-ing errors? Else we would need to configure that separately.